README.md

```diff
@@ -15,30 +17,32 @@ processes/machines. It is also a generic Java Agent and could be used for any JV
 ## How to Build
 
 1. Make sure JDK 8+ and maven is installed on your machine.
-2. Run: mvn clean package
+2. Run: `mvn clean package`
+
+This command creates **jvm-profiler-1.0.0.jar** file with the default reporters like ConsoleOutputReporter, FileOutputReporter and KafkaOutputReporter bundled in it. If you want to bundle the custom reporters like RedisOutputReporter or InfluxDBOutputReporter in the jar file then provide the maven profile id for that reporter in the build command. For example to build a jar file with RedisOutputReporter, you can execute `mvn -P redis clean package` command. Please check the pom.xml file for available custom reporters and their profile ids. 
```

Comment:
we should leave out the specific version number since they are updated frequently? `-1.0.0`

Reply:
Thank you Felix for quick review. This is the same travis badge with black and green boxes for build status. Is there some other way? Please let me know. Version numbers are there with all the sample commands given in this readme page. If we remove the version number then the new user might get an error if he directly copy and paste the command from this page.

Reply:
If I use `**jvm-profiler-*.jar**`, markdown editor will show it as `jvm-profiler-.jar*` so I am keeping it as `**jvm-profiler.jar**`.